starter_code/models/movie.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+
+const Schema = mongoose.Schema;
+
+
+const movieSchema = new Schema ({
+  title : {type : String, required : true},
+  director : {type : String, required : true},
+  stars : [String],
```

Comment:
this should be 

```
stars : Array,
showtimes : Array
```

instead of String. If you look at the data in your seed file, you can see that stars and showtimes are both arrays